src/facade/dragonfly_connection.cc

```diff
@@ -514,6 +514,25 @@ void Connection::OnPostMigrateThread() {
     socket_->RegisterOnErrorCb([this](int32_t mask) { this->OnBreakCb(mask); });
     break_cb_engaged_ = true;
   }
+
+  auto update_tl_vars = [this] [[gnu::noinline]] {
```

Comment:
you do not need this lambda anymore. OnPostMigrateThread is virtual function being called from helio.
So it is not possible that the code here is inlined.